.github/workflows/continuous-integration.yml

```diff
@@ -121,15 +161,39 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-    - name: Install opensim-core
-      run: |
-        wget https://prdownloads.sourceforge.net/myosin/opensim-core/opensim-core-latest_linux_Release.zip
-        # -q: quiet
-        unzip -q opensim-core-latest_linux_Release.zip -d ~
 
     - name: Checkout opensim-gui
       uses: actions/checkout@v1
 
+    
+    - name: Checkout opensim-core master
+      uses: actions/checkout@v2
+      with:
+         name: opensim-org/opensim-core
+         repository: opensim-org/opensim-core
+         path: 'opensim-core'
+    
+    - name: Build opensim-core-dependencies
+      run: |
+        echo $env:GITHUB_WORKSPACE\\build_deps
```

Comment:
Commands run from the GITHUB_WORKSPACE folder by default; I also don't believe the `env:` is necessary, `$GITHUB_WORKSPACE` should work.
Also, the path separator on Linux is forward-slash `/`.